src/main/resources/bundleconfiguration/iac-remitted-reheard-hearing-bundle-config.yaml

```diff
@@ -12,10 +12,43 @@ sort:
 hasFolderCoversheets: true
 pageNumberFormat: numberOfPages
 folders:
+  - name: Appellant additional evidence following FTPA
+    documents:
+      - type: documentSet
+        property: /appAdditionalEvidenceDocs
+  - name: Respondent additional evidence following FTPA
+    documents:
+      - type: documentSet
+        property: /respAdditionalEvidenceDocs
   - name: Appellant FTPA documents
     documents:
       - type: documentSet
         property: /ftpaAppellantDocuments
+  - name: Respondent FTPA documents
+    documents:
+      - type: documentSet
+        property: /ftpaRespondentDocuments
+  - name: Decisions and reasons documents
```

Comment:
We have 'Decisions and reasons documents' and 'Reheard hearing documents' already?